src/js/base/stories/LinkButton.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { LinkButton } from "../Button";
+
+export default {
+    title: "base/Button/LinkButton",
+    component: LinkButton
+};
+
+const Template = args => <LinkButton {...args} />;
+
+export const greenLinkButton = Template.bind({});
+
+greenLinkButton.args = {
+    color: "green",
+    to: "/",
```

Comment:
This should keep us at the same path instead of going back to root.
```suggestion
    to: "#",
```